contractcourt/commit_sweep_resolver_test.go

```diff
@@ -192,14 +192,20 @@ func TestCommitSweepResolverDelay(t *testing.T) {
 
 	ctx := newCommitSweepResolverTestContext(t, &res)
 
+	// Setup whether we expect the sweeper to receive a sweep error in this
+	// test case.
+	ctx.sweeper.sweepErr = sweepErr
+
 	report := ctx.resolver.report()
-	if !reflect.DeepEqual(report, &ContractReport{
+	expectedReport := ContractReport{
 		Outpoint:     outpoint,
 		Type:         ReportOutputUnencumbered,
 		Amount:       btcutil.Amount(amt),
 		LimboBalance: btcutil.Amount(amt),
-	}) {
-		t.Fatal("unexpected resolver report")
+	}
+	if *report != expectedReport {
```

Comment:
I think it's safer to use `DeepEqual` here (even this is currently fine).

Reply:
I actually suggested to do the direct comparison. I think both work.

Reply:
I agree, in this case both work just fine. Perhaps we can keep direct comparison as it's much faster.